pikaur/build.py

```diff
@@ -102,8 +117,12 @@ class PackageBuild(DataType):
 
     def __init__(self, package_name):  # pylint: disable=super-init-not-called
         self.package_name = package_name
-        self.build_dir = os.path.join(BUILD_CACHE, self.package_name)
-        repo_path = os.path.join(AUR_REPOS_CACHE, package_name)
+        if asroot():
+            self.build_dir = "/var/cache/pikaur/build"
```

Comment:
better will be ROOT_BUILD_CACHE_DIR and USER_CACHE_DIR in `pikaur/config.py`

Reply:
I'm not sure we want that in `config.py`, because we can't configure that path. `/var/cache/<something>` is the path given by systemd-run -p CacheDirectory=<something>, which can't really be changed easily.

Unless you mean ROOT_BUILD_CACHE_DIR should just contain `build` and `aur_packages`? That could also work.

Reply:
yup, latter looks fine
anyway it would be better to avoid using strings right in code

yes, i know there are quite a lot of such places in the code already but i am trying to reduce them when noticing such places and so replace them either to some "constants" on top of file or in some separate module